ui/server/graphql/resolvers/index.ts

```diff
@@ -259,6 +261,30 @@ const resolvers = {
         return null;
       }
     },
+    roundInvitationLink: async (parent, { roundId }, { user }) => {
+      const isAdmin =
+        (await !!user) &&
+        isCollAdmin({
+          userId: user?.id,
+          roundId,
+        });
+
+      if (!isAdmin) {
+        throw new Error("You need to be admin to fetch invitation link");
+      }
+
+      const round = await prisma.round.findFirst({
+        where: {
+          id: roundId,
+        },
+      });
+      return {
+        link:
```

Comment:
Why is this nested?

Reply:
to match schema

Reply:
yeah i know i was wondering if we could simplify the schema. But no big deal, you can 'Resolve' this if you want.

Reply:
I think it is ok to keep like this. If in future adding more field to this type is required, it would be easy to handle the changes.